### PR TITLE
updated css so headers with two lines dont look weird

### PR DESCRIPTION
**Pull request by im2nguyen**
- update css so headers with two lines don't look weird
- assuming that the line-height is 44px, since it's defined in the css

current:
image

proposed: 
image

single line:
image

### CODE REVIEW

**Comment by im2nguyen** [comment]
@nfagerlund, I commented out margin-top: 0 if grid is supported for both the sidebar and the main content, which produces the desired result (everything is just shifted down 20px). please let me know if this works. if not, the other route is to change the h1 font size to 2.25 rem, which also produces the desired result

image

image

File: content/Gemfile.lock [hunk 1]
```diff
@@ -15,7 +15,7 @@ GEM
     capybara (2.4.4)
       mime-types (>= 1.16)
       nokogiri (>= 1.3.3)
-      rack (>= 1.0.0)
+      rack (>= 1.6.12)
```
Comment (nfagerlund) [inline content/Gemfile.lock]: Please revert all the gemfile changes -- those are fairly disruptive and require building and shipping a new Docker container, so they're not something that can be done as an aside in an unrelated PR.

File: content/source/assets/stylesheets/_docs.scss [hunk 2]
```diff
@@ -37,7 +37,7 @@
 
     margin-top: 20px; // fallback for non-grid browsers
     @supports (display: grid) {
-      margin-top: 0;
+      // margin-top: 0; // commented out to account for two row headers
```
Comment (nfagerlund) [inline content/source/assets/stylesheets/_docs.scss]: The right way to do this is to remove these whole `@supports` query blocks, and also remove the "only for non-grid" comments above. But I approve the general approach of just leaving some extra space. 👍🏼

Comment (im2nguyen) [reply content/source/assets/stylesheets/_docs.scss]: yep, i would have removed all the unnecessary code, but wasn't sure if we were going to need it in the future, seeing how this is just a temp fix 

```
 // fallback for non-grid browsers
    @supports (display: grid) {
      // margin-top: 0; // commented out to account for two row headers
    }
```

File: content/Gemfile.lock (hunk 1 above)
Comment (im2nguyen) [reply content/Gemfile.lock]: sounds good, new commit will revert this change. will be happy to pair with you when we do choose to update the gem!

**Pull request merged**